Kconfig

```diff
@@ -40,6 +40,10 @@ config BOOT_LOADER
 	bool
 	default n
 
+config HAVE_RESET_VECTOR_ROM
+	bool
+	default n
+
```

Comment:
I know it's hidden symbol, but it would be nice to have `help` to describe a bit what it is.

Reply:
@jajanusz will fix in v2.

Reply:
@dbaluta Thanks